src/plays/age-calculator-by-react/AgeCalculatorByReact.js

```diff
@@ -0,0 +1,23 @@
+import PlayHeader from 'common/playlists/PlayHeader';
+import './styles.css';
+import Calculator from './CalculatorHeader';
+
+// WARNING: Do not change the entry componenet name
+function AgeCalculatorByReact(props) {
+  // Your Code Start below.
+
+  return (
+    <>
+      <div className="play-details">
+        <PlayHeader play={props} />
+        <div className="play-details-body">
```

Comment:
Add a unique style here beside `play-details-body` and add that as prefix to all the styles in styles.css file.

Reply:
Add play-details-body css